gravitee-apim-console-webui/package.json

```diff
@@ -20,11 +20,16 @@
     "@asyncapi/web-component": "1.2.26",
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2024-43796   
**Check Name:** express: Improper Input Handling in Express Redirects   
**Severity:** MEDIUM   
**Fixed Version:** 4.20.0, 5.0.0   
**Reachable Path(s) Found:** No  
**Description:** Express.js minimalist web framework for node. In express < 4.20.0, passing untrusted user input - even after sanitizing it - to response.redirect() may execute untrusted code. This issue is patched in express 4.20.0.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2024-43796